ios-drag-drop.js

```diff
@@ -316,7 +318,8 @@
   }
 
   function once(el, event, handler, context) {
-    if(context) handler = handler.bind(context)
+    if(context)
+      handler = handler.bind(context);
```

Comment:
either, just not the error prone style :)

On Sat, Aug 15, 2015 at 9:19 AM, Jonathan Dumaine <notifications@github.com>
wrote:

> In ios-drag-drop.js
> <https://github.com/timruffles/ios-html5-drag-drop-shim/pull/45#discussion_r37134530>
> :
>
> > @@ -316,7 +318,8 @@
> >    }
> >
> >    function once(el, event, handler, context) {
> > -    if(context) handler = handler.bind(context)
> > +    if(context)
> > +      handler = handler.bind(context);
>
> Which would you prefer? Single line w/ semicolon or brackets? I thought it
> was a mistake at first with it being an if statement on a single line with
> no semicolon.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/timruffles/ios-html5-drag-drop-shim/pull/45/files#r37134530>
> .
>